### ISSUE TITLE
New image cropping 

### ISSUE THREAD

**Issue by maskas** [impresspages/ImpressPages#658]
There is a discussion to replace current image cropping script with one of these: http://www.jqueryrain.com/demo/jquery-crop-image-plugin/

What do you think?

**Comment by maskas** [comment]
These looks like the best to me:
http://www.jqueryrain.com/?BEAlLLl9
http://www.jqueryrain.com/?jKrB7Iva

**Comment by maverikt** [comment]
I like this one
http://www.jqueryrain.com/?jKrB7Iva
Tony

**Comment by byybora** [comment]
Best photo editor I've ever seen. 

https://www.aviary.com/

Demo Here: https://developers.aviary.com/features

**Comment by maskas** [comment]
It is not free right?

**Comment by byybora** [comment]
I see so many times in some scripts. I didnt know but i guess basic edition is free?

https://developers.aviary.com/pricing

**Comment by maskas** [comment]
1 megapixel output for Web. Contact us for full resolution support (up to 250,000 saves/month is free).
that probably means aviary could be enabled as an add-on. For those who don't mind to enter their Aviary credentials before being able to use it.

**Comment by eazuka** [comment]
The Aviary SDK is now part of the Adobe Creative SDK. Aviary’s basic SDK is powerful, proven and completely free. I think we should focus on selecting from the list you provided

**Comment by eazuka** [comment]
@maskas, i think your 2 choices above are nice. we just have to select 1 of the 2.
it seem http://www.jqueryrain.com/?BEAlLLl9 has more feature and kind of more flexible.

**Comment by rzusto** [comment]
I agree with @eazuka. Rotate feature is important. I always missed this function. Sometimes photos uploads from smartphones are switched 90 degree.

**Comment by feniweb** [comment]
Actually "imp" has already a cropping function, actually it's missing is the size specified for the image.

As I see on my screen print. And I think the editor is not so confusing whichhe does not understand.
See image with a proposal. 2 input fields for height and width. Are great.
![Image 2 input fields](http://d4m.ch/imp/file/2014/12/10/Zuschneiden_2__2.jpg)

I have looked at Neos Typo3 and there is the änlich dismounting it with an input image size.

Thank you for your great effort for a simple CMS.

**Comment by eazuka** [comment]
Hi @maskas, any update on this or has this idea been dropped?

**Comment by maskas** [comment]
It has been dropped due to the lack of resources for new development.
